src/iperf_api.c

```diff
@@ -4613,6 +4648,16 @@ iperf_common_sockopts(struct iperf_test *test, int s)
             }
         }
     }
+
+#if defined(HAVE_SO_PRIORITY)
+    if ((opt = test->settings->sock_prio)) {
```

Comment:
This may be a non-issue, this statement means that if the user set the priority parameter to 0, `sectsockopt()` will not be executed.  Although 0 is the default priority, the user may want for some reason that the `setsockopt()` will be executed with priority 0.  Therefore, it may be better that `sock_prio` will be set to `-1` (note that if this is done then it has to be taken into account in changes like the `cJSON_AddItemToObject()` change in `iperf_on_test_start()`).